src/lib.rs

```diff
@@ -40,10 +40,10 @@ extern crate graphene;
 extern crate gsk4 as gsk;
 extern crate pango;
 
-#[cfg(feature = "futures")]
+#[cfg(any(feature = "futures", feature = "dox"))]
 extern crate fragile;
-#[cfg(feature = "futures")]
-extern crate futures_core;
+#[cfg(any(feature = "futures", feature = "dox"))]
```

Comment:
```suggestion
#[cfg(feature = "futures")]
```

Same here. "futures" is implied by "dox" in Cargo.toml.

Reply:
Done, thanks